content/04-class-field.md

```diff
@@ -333,7 +333,16 @@ Main.main();
 })({});
 ```
 
+Note that even though we call such kind of fields "variables", inline variables can never
```

Comment:
I don't think text should be manually wrapped, to be consistent with the rest of the manual.